src/misc/dispatcher.ts

```diff
@@ -0,0 +1,14 @@
+export function dispatch(
+	eventName: string,
+	fileName: string,
+  data?: any,
+): void {
+	const eventMonitor = document.getElementById('eventMonitor');
+	if (!eventMonitor) {
+		console.log(`eventMonitor not found to dispatch event ${eventName} for ${fileName}`);
```

Comment:
Since this is in the library itself, please replace `console.log` with `log` and use:

``` ts
import {  log } from "./logger";
```

Allows people to `show/hide` logging